velox/substrait/VeloxToSubstraitType.cpp

```diff
@@ -158,11 +158,29 @@ const ::substrait::Type& VeloxToSubstraitTypeConvertor::toSubstraitType(
       substraitType->set_allocated_struct_(substraitStruct);
       break;
     }
+    case velox::TypeKind::DATE: {
+      ::substrait::Type_Date* subDate =
+          google::protobuf::Arena::CreateMessage<::substrait::Type_Date>(
+              &arena);
+      subDate->set_nullability(
+          ::substrait::Type_Nullability_NULLABILITY_NULLABLE);
+      substraitType->set_allocated_date(subDate);
+      break;
+    }
     case velox::TypeKind::UNKNOWN: {
+      // velox unknown type binding to substrait unknown type
```

Comment:
Comments need to be full sentences. Start with a capital letter and end with a period. Please, fix throughout.